R/survival.R

```diff
@@ -9,30 +9,32 @@
-#' @param .large parameter for large sized fish, source: \href{https://afspubs.onlinelibrary.wiley.com/doi/abs/10.1577/M02-161.1}{Connor et al. (2004)}
-#' @param .floodplain parameter for floodplain rearing benefit, source: \href{https://dsm-docs.s3-us-west-2.amazonaws.com/SOMMER_T-SDWA+180+Floodplain+rearing+of+juvenile+chinook+salmon+evidence+of+enhanced+growth+and+survival+.pdf}{Sommer et al. (2001)}
+#' @param .avg_temp_thresh coefficient for avg_temp_thresh variable
+#' @param .high_predation coefficient for high_predation variable
+#' @param ..surv_juv_rear_contact_points coefficient for contact_points variable
+#' @param ..surv_juv_rear_prop_diversions coefficient for prop_diversions variable
+#' @param ..surv_juv_rear_total_diversions coefficient for total_diversions variable
+#' @param .stranded coefficient for stranded variable
+#' @param .medium parameter for medium sized fish
+#' @param .large parameter for large sized fish
+#' @param .floodplain parameter for floodplain rearing benefit
+#' @param min_survival_rate estimated survival rate if temperature threshold is exceeded, source: expert opinion
 #' @source IP-117068
 #' @export
 surv_juv_rear <- function(max_temp_thresh, avg_temp_thresh, high_predation,
                           contact_points, prop_diversions, total_diversions,
                           stranded, weeks_flooded,
-                          ..surv_juv_rear_int = 3.5,
-                          .avg_temp_thresh = -0.717,
-                          .high_predation = -0.122,
-                          ..surv_juv_rear_contact_points = -0.0068,
-                          ..surv_juv_rear_prop_diversions = -0.1755,
-                          ..surv_juv_rear_total_diversions = -0.0005,
-                          .stranded = -1.939,
-                          .medium = 1.48,
-                          .large = 2.223,
-                          .floodplain = 0.47){
+                          ..surv_juv_rear_int = fallRunDSM::params$..surv_juv_rear_int,
+                          .avg_temp_thresh = fallRunDSM::params$.surv_juv_rear_avg_temp_thresh,
+                          .high_predation = fallRunDSM::params$.surv_juv_rear_high_predation,
+                          ..surv_juv_rear_contact_points = fallRunDSM::params$..surv_juv_rear_contact_points,
+                          ..surv_juv_rear_prop_diversions = fallRunDSM::params$..surv_juv_rear_prop_diversions,
+                          ..surv_juv_rear_total_diversions = fallRunDSM::params$..surv_juv_rear_total_diversions,
+                          .stranded = fallRunDSM::params$.surv_juv_rear_stranded,
+                          .medium = fallRunDSM::params$.surv_juv_rear_medium,
+                          .large = fallRunDSM::params$.surv_juv_rear_large,
+                          .floodplain = fallRunDSM::params$.surv_juv_rear_floodplain,
+                          min_survival_rate){
```

Comment:
i added `min_survival_rate`, looking at the others I think that the argument signature should include `fallRunDSM::params$min_survival_rate` as the default value. seems like it would be consistent with `surv_egg_to_fry`. for elements that will be included in the sensitivity analysis, but are not intercepts or coefficients, what do you think is the right approach?

Reply:
Agreed. Added in for all submodules that use `min_survival_rate`.